src/DependencyInjection/FOSElasticaExtension.php

```diff
@@ -94,10 +94,10 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadClients($config['clients'], $container);
         $container->setAlias('fos_elastica.client', \sprintf('fos_elastica.client.%s', $config['default_client']));
-        $container->getAlias('fos_elastica.client')->setPublic(true);
+        $container->getAlias('fos_elastica.client')->setPrivate(true);
```

Comment:
`setPrivate()` should not be used since Symfony 3.4, use `setPublic(false)`
See https://github.com/symfony/symfony/pull/37373